src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
+  const headers = table.querySelectorAll('th');
+  const rows = Array.from(table.querySelectorAll('tbody tr'));
+
+  const sortRows = (index) => {
+    const isNumeric = index === 2 || index === 3;
+
+    rows.sort((a, b) => {
+      const cellA = a.cells[index].textContent.trim();
+      const cellB = b.cells[index].textContent.trim();
+
+      if (isNumeric) {
+        return (
+          parseFloat(cellA.replace(/[$,]/g, '')) -
+          parseFloat(cellB.replace(/[$,]/g, ''))
```

Comment:
The numeric sorting logic assumes that all values in the specified columns are valid numbers after removing `$` and `,`. If there are any non-numeric values, `parseFloat` will return `NaN`, which can lead to unexpected sorting behavior. Consider adding a check to handle non-numeric values gracefully.